public/app/features/explore/RichHistory/RichHistoryQueriesTab.tsx

```diff
@@ -248,6 +252,11 @@ export function RichHistoryQueriesTab(props: Props) {
               </div>
             );
           })}
+        {queries.length && queries.length !== totalQueries ? (
+          <div>
+            Showing {queries.length} of {totalQueries}. <Button onClick={loadMoreRichHistory}>Load more</Button>
```

Comment:
Nit: no period at the end of this

Reply:
Fixed in 1bce13cf888ecfdf5fb12e6929c66137a6998a10